README.md

```diff
@@ -16,7 +16,8 @@ describing the target cloud infrastructure.
 | GCloud   | Compute Engine - VM      |:white_check_mark:|
 | GCloud   | CloudSQL                 |:white_check_mark:|
 | GCloud   | AlloyDB                  |:white_check_mark:|
-| Azure    | VM                       |:white_check_mark:|
+| GCloud   | Google Kubernetes Engine |:white_check_mark:|
+| Azure    | VM                       |       :x:        |
```

Comment:
This file is undoing #16 merged changes.
`azure` is supported

Reply:
Updated